src/database/migrations/20210106143948-create-positions-table.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("positions", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.TINYINT,
+      },
+      name: {
+        type: Sequelize.STRING(30),
```

Comment:
- Think this is too small, make it a little bit bigger, maybe 100 chars because 30 is a bit too small, maybe there will be a position in the future that will have more than 30 chars.